loader/src/producer_class.py

```diff
@@ -48,6 +52,11 @@ def produce_event(self, topic, record):
             self.producer.poll(1)  # Callback function
         except ValueError:
             logger.error(format_traceback())
+        except KafkaException as e:
+            if e.args[0].code() == KafkaError.UNKNOWN_TOPIC_OR_PART:
+                pass
```

Comment:
Same

Reply:
I'm ignoring this because It's expected when we cancel a batch (the topic is deleted).